### PR TITLE
{toolchain} goblf/2020b (incl. LAPACK 3.9.0)

### PR DESCRIPTION
**Pull request by boegel**
(created using `eb --new-pr`)
requires ~~https://github.com/easybuilders/easybuild-easyblocks/pull/2358~~ (to compile LAPACK 3.9.0 with correct flags)

### CODE REVIEW

**Comment by boegel** [comment]
@boegelbot please test @ generoso
EB_ARGS="--include-easyblocks-from-pr 2358"

**Comment by smoors** [comment]
Test report by @smoors
**SUCCESS**
Build succeeded for 3 out of 3 (3 easyconfigs in total)
node381.hydra.os - Linux centos linux 7.9.2009, x86_64, Intel(R) Xeon(R) Gold 6148 CPU @ 2.40GHz (skylake_avx512), Python 2.7.5
See https://gist.github.com/e1051954e9ab16153fc6b59719ecb936 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 3 out of 3 (3 easyconfigs in total)
node3507.doduo.os - Linux RHEL 8.2, x86_64, AMD EPYC 7552 48-Core Processor (zen2), Python 3.6.8
See https://gist.github.com/f79e5a5118d1313b0db144505393023d for a full test report.

**Comment by boegel** [comment]
@boegelbot please test @ generoso

File: easybuild/easyconfigs/s/ScaLAPACK/ScaLAPACK-2.1.0-gompi-2020b-bl.eb [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+    ('BLIS', '0.8.0'),
+    ('LAPACK', '3.9.0'),
+]
+
+# Config Opts based on AOCL User Guide:
+# https://developer.amd.com/wp-content/resources/AOCL_User%20Guide_2.2.pdf
+
+configopts = '-DBUILD_SHARED_LIBS=ON '
+configopts += '-DBLAS_LIBRARIES="$EBROOTBLIS/lib/libblis.a" '
+configopts += '-DLAPACK_LIBRARIES="$EBROOTLIBFLAME/lib/libflame.a" '
```
[the remark this answers is not in the document]
Comment (boegel) [reply easybuild/easyconfigs/s/ScaLAPACK/ScaLAPACK-2.1.0-gompi-2020b-bl.eb]: Of course, fixed in 785231fb57

**Comment by boegel** [comment]
@boegelbot please test @ generoso

**Comment by boegel** [comment]
Test report by @boegel
Using easyblocks from PR(s) https://github.com/easybuilders/easybuild-easyblocks/pull/2358
**SUCCESS**
Build succeeded for 4 out of 4 (4 easyconfigs in total)
node3125.skitty.os - Linux centos linux 7.9.2009, x86_64, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz, Python 3.6.8
See https://gist.github.com/d6c8dc246bae9ad9a5835d2ee562a358 for a full test report.

**Comment by boegel** [comment]
@boegelbot please test @ generoso
EB_ARGS="--include-easyblocks-from-pr 2358"

**Comment by migueldiascosta** [comment]
Test report by @migueldiascosta
**SUCCESS**
Build succeeded for 4 out of 4 (4 easyconfigs in total)
sms - Linux centos linux 7.6.1810, x86_64, AMD EPYC 7601 32-Core Processor (zen), Python 2.7.5
See https://gist.github.com/6df00dc5b38c0ddbe159d790dfbada91 for a full test report.

**Review by migueldiascosta — APPROVED** [review]
lgtm

**Comment by migueldiascosta** [comment]
Going in, thanks @boegel!

**Comment by akesandgren** [comment]
Since LAPACK 3.9.1 is out we should perhaps use that instead...

**Comment by boegel** [comment]
@akesandgren done in #12535

**Pull request merged**